### DISCUSSION TITLE
osu! crashes when trying to play the 2nd map in current spotlights playlist

### DISCUSSION THREAD

**Discussion by respektive** [ppy/osu#17035] [Q&A] [answered]
Beatmap that causes the crash: https://osu.ppy.sh/beatmapsets/1560870#osu/3235071

Whenever i try to start that map, osu! crashes with the error `malloc(): unsorted double linked list corrupted` in the console. I noticed that the beatmap has a video and runtime logs show something about the video being too far out of sync. I tried turning off hardware acceleration but that didn't help.

logs:
[database.log](https://github.com/ppy/osu/files/8164654/database.log)
[network.log](https://github.com/ppy/osu/files/8164655/network.log)
[performance.log](https://github.com/ppy/osu/files/8164656/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8164657/runtime.log)

**Comment by bdach** [comment, marked as answer]
Cannot reproduce.

Please post output of `ffmpeg --version`.

**Reply by respektive** [reply]
```
ffmpeg version n5.0 Copyright (c) 2000-2022 the FFmpeg developers
built with gcc 11.2.0 (GCC)
configuration: --prefix=/usr --disable-debug --disable-static --disable-stripping --enable-amf --enable-avisynth --enable-cuda-llvm --enable-lto --enable-fontconfig --enable-gmp --enable-gnutls --enable-gpl --enable-ladspa --enable-libaom --enable-libass --enable-libbluray --enable-libdav1d --enable-libdrm --enable-libfreetype --enable-libfribidi --enable-libgsm --enable-libiec61883 --enable-libjack --enable-libmfx --enable-libmodplug --enable-libmp3lame --enable-libopencore_amrnb --enable-libopencore_amrwb --enable-libopenjpeg --enable-libopus --enable-libpulse --enable-librav1e --enable-librsvg --enable-libsoxr --enable-libspeex --enable-libsrt --enable-libssh --enable-libsvtav1 --enable-libtheora --enable-libv4l2 --enable-libvidstab --enable-libvmaf --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxcb --enable-libxml2 --enable-libxvid --enable-libzimg --enable-nvdec --enable-nvenc --enable-shared --enable-version3
libavutil      57. 17.100 / 57. 17.100
libavcodec     59. 18.100 / 59. 18.100
libavformat    59. 16.100 / 59. 16.100
libavdevice    59.  4.100 / 59.  4.100
libavfilter     8. 24.100 /  8. 24.100
libswscale      6.  4.100 /  6.  4.100
libswresample   4.  3.100 /  4.  3.100
libpostproc    56.  3.100 / 56.  3.100
```

**Reply by bdach** [reply]
Works fine on 4.3.2 here so I presume this may be a fresh upstream bug in FFmpeg or even [something related to breaking changes](https://github.com/ppy/osu-framework/issues/5051). Linux release does not bundle FFmpeg libs in the appimage yet so for now it will use the system-wide install.

Either way 99% something to isolate and report upstream.

**Reply by respektive** [reply]
Seems to be an issue with that specific ffmpeg version. Updated to latest development branch and the issue is gone. I will mark this discussion as answered.
